computations.py

```diff
@@ -0,0 +1,140 @@
+import calendar
+import sys
+
+
+class Computations:
+
+    @staticmethod
+    def get_weather_attribute_index(weather_attributes_list, attribute):
+        return weather_attributes_list[attribute]
+
+    @staticmethod
+    def get_month_and_day_from_date(date):
+        year, month, day = date.split("-")
+        return int(month), int(day)
+
+    @staticmethod
+    def find_maximum_temperature_in_year(year, all_weather_records):
+        weather_attributes, weather_records = all_weather_records
+        maximum_temperature = Computations.get_weather_attribute_index(weather_attributes, "Max TemperatureC")
+        date = Computations.get_weather_attribute_index(weather_attributes, "PKT")
+        maximum_temperature_in_year = float("-inf")
+        maximum_temperature_record_in_year = []
+        try:
```

Comment:
If you have a function having length of more than 5 lines
You should be using blank spaces above try-except, if-else or loops in order to increase readability.

Reply:
Sure